module/icp/io/sha2_mod.c

```diff
@@ -591,7 +591,7 @@ sha2_digest_atomic(crypto_provider_handle_t provider,
 {
 	int ret = CRYPTO_SUCCESS;
 	SHA2_CTX sha2_ctx;
-	uint32_t sha_digest_len;
+	uint32_t sha_digest_len = SHA256_DIGEST_LENGTH;
```

Comment:
Initialization is not needed, this should be dropped.  It can never be accessed uninitialized.